src/SprykerMiddleware/Zed/Process/ProcessConfig.php

```diff
@@ -6,4 +6,35 @@
 
 class ProcessConfig extends AbstractBundleConfig
 {
+    /**
+     * @return array
```

Comment:
Please provide the specification for these config methods, so far you have a special structure for them.
You could add an `@example` into the phpdoc-s.